code/index.html

```diff
@@ -56,6 +63,13 @@ <h1>Does the person have</h1>
           alt="Guess Who"
         />
         <h1 id="winOrLoseText"></h1>
+        <div class="game-board" id="secret-person-reveal">bjeb</div>
+        <!-- <img
+          id="secret-image"
+          class="secret-image"
+          src="images/guess-who-bubble.png"
+          alt="Secret"
+        /> -->
```

Comment:
A small thing but maybe delete what you have commented out and don't want to use.

Reply:
Yes, remember to always clean your code from all console.log()s and commented out code blocks and tips left for you by teachers 🗡️ ✂️